### PR TITLE
ios sdk

### PR DESCRIPTION
**Pull request by jishnu7**
This change is [Reviewable](https://reviewable.io/reviews/hashcube/hotline/2)

### CODE REVIEW

**Comment by rampr** [comment]
Review status: 0 of 577 files reviewed at latest revision, 1 unresolved discussion.

---

_[a discussion](https://reviewable.io:443/reviews/hashcube/hotline/2#-KPu4_VPra9S-vvLoFm6:-KPu4_VPra9S-vvLoFm7:b-lf5s3) (no related file):_
Both iOS and android are present, please merge android first. Will be easy to review

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/hashcube/hotline/2)_

**Comment by jishnu7** [comment]
Review status: 0 of 577 files reviewed at latest revision, 1 unresolved discussion.

---

_[a discussion](https://reviewable.io:443/reviews/hashcube/hotline/2#-KPu4_VPra9S-vvLoFm6:-KPu5HDQYr9xZyH9mM9v:b-97esu) (no related file):_
<details><summary>

<i>Previously, rampr (ram) wrote…</i></summary>



> Both iOS and android are present, please merge android first. Will be easy to review
> </details>
> Changed target branch. 

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/hashcube/hotline/2)_

File: js/index.js [hunk 1]
```diff
@@ -56,6 +56,9 @@ exports = new (Class(function () {
   };
 
   this.addMetaData = function (name, value) {
+    if (typeof value !== "string") {
+      value = JSON.stringify(value);
```
Comment (rampr) [inline js/index.js:60]: what if it is not string and not object

Comment (jishnu7) [reply js/index.js:60]: if it is integer stringify will convert it to string.

**Comment by bijoshtj** [comment]
_[android/config.json, line 1 [r1]](https://reviewable.io:443/reviews/hashcube/hotline/2#-KPulS6ivRuWrqibQTRG:-KPulS6ivRuWrqibQTRH:b3ecddx) ([raw file](https://github.com/hashcube/hotline/blob/390a90b3af61c391a9220c87ed4612786432cc9f/android/config.json#L1)):_

> ``` JSON
> {
> ```

is this missed in android pr?

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/hashcube/hotline/2)_

File: js/index.js (hunk 1 above)
Comment (rampr) [reply js/index.js:60]: 👍

**Comment by rampr** [comment]
LGTM

**Comment by bijoshtj** [comment]
LGTM

**Pull request merged**